### PR TITLE
Add current account team visual

### PR DESCRIPTION
**Pull request by Iruuka**
Refers to Issue https://github.com/aMytho/brave-frontier-godot/issues/9

What is done (for now) with this commit : 
- Get first team found of user X in database
- Get each full-sprite of the team's unit
- Show them in the lobby menu
- Show the good element symbol to each character (fire symbol for fire unit for example)

What NEED to be done (and don't actually or doesn't work fine) : 
- Show the unit's sprite without skretching or making them weird

What is fix overall : 
- For the background of the element, the thunder element was refering to file as "lightning_[number]". Problem is that we get the background using element of unit. And for those unit, we use "thunder" instead of "lightning". So we couldn't get the background right
- For the file "home.tscn", the element "char3" and "char5" were refering, for each, and in that order, "char5" and "char3". I fix it so the good name was used for each

For an example of the actual work : https://drive.google.com/file/d/1B0IDU1v2Mn_GYIXRJeQH_X8z5axR3dkb/view?usp=share_link
**Important to test with new account :** you MUST wait until the fight start for your team and the 2 slime are saved. So you must Thilith has finish talking

### CODE REVIEW

**Comment by Iruuka** [comment]
For today, i'll try to add the element symbol for each type of character. I'll push a prototype if I can do one good. If not, nothing will change.

Also, it looks like the value is set in code. So i'll report the image region of each symbol and update them

**Comment by Iruuka** [comment]
Hello @aMytho !

For this pull request, you can base yourself on the description of what has been done.
You can also see an actual visual using the google drive link.

The only problem here is the stretching png. I didn't try too much to correct it (because as it is, it doesn't block the user).
I let you review the code. If you find it messy or need to be rewrite, tell me anytime ! (I'll try to do some when I can)

If you're fine  with the unit stretching, i open the pull request !

Have a good day/night !

File: Menu/SubMenu/Home/home.gd [hunk 1]
```diff
@@ -1,10 +1,60 @@
...
+	
+	var charElementList = [
+		$Character/TextureRect/Char1/CharElement,
+		$Character/TextureRect/Char2/CharElement,
+		$Character/TextureRect/Char3/CharElement,
+		$Character/TextureRect/Char4/CharElement,
+		$Character/TextureRect/Char5/CharElement
+	]
+	
+	var unitDataResources = load("res://Database/resource_lookups.gd").new();
```
Comment (aMytho) [inline Menu/SubMenu/Home/home.gd]: You can use the autoload `Lookups` here. It is already instantiated so we don't have to make a new instance of it.

Ex. `Lookups.get_first_team_by_account_id(ActiveAccount.id)`

**Review by aMytho — COMMENTED** [review]
Looks good. I'm not worried about the scaling, we can figure that out later. Its probably some random property that was checked accidentally. The `main` branch has a similar issue with the slimes being smaller than the other units.

I made one suggestion which can be seen in the review.

The only other thing I can think of is to load the default background if a unit is missing. It doesn't have to be in this PR though.

(If you want to try it, its in the main asset dump `assets/content/home/home_character_frame_bg`. Link is [here](https://github.com/aMytho/brave-frontier-godot/issues/2#issuecomment-1498443641). Its a very large collection though!

**Comment by Iruuka** [comment]
Hello @aMytho 

I've taken into account your review.
If everything is fine by you, I let you merge it !
Here's a pick of the work : 
image

Have a good day/night !

**Comment by aMytho** [comment]
Looks great, thanks!

I'll see if I can get the background animations working again. The turn system is getting pretty complex so I might switch to a UI task for a bit.

Edit: I just realized they were still working. I forgot how subtle they were. I'll need to look at the original to compare them at some point.

**Pull request merged**